lib/active_record/vendor/odbcext_postgresql.rb

```diff
@@ -82,7 +82,7 @@ def quoted_false
 
   def quote_string(string)
     @logger.unknown("ODBCAdapter#quote_string>") if @trace
-    string.gsub(/\\/, '\&\&').gsub(/'/, "''")
+    string.gsub(/\\/, '\&').gsub(/'/, "''")
```

Comment:
This was taken from master of rails-localytics. Note there are two commits in this PR, the first is applying local changes form localytics-rails.